Wikipedia/Code/ArticleLocationAuthorizationCollectionViewCell.swift

```diff
@@ -41,9 +41,6 @@ class ArticleLocationAuthorizationCollectionViewCell: ArticleLocationExploreColl
     }
```

Comment:
message: 

Make the check in `ExploreCardViewController` instead of accessing `WMFLocationManager` from the cell.